dipy/workflows/reconst.py

```diff
@@ -1032,3 +1033,193 @@ def get_fitted_ivim(self, data, mask, bval, bvec, b0_threshold=50):
+            Threshold used to find b0 volumes.
+        bvecs_tol : float, optional
+            Bvecs should be unit vectors.
+        roi_center : variable int, optional
+            Center of ROI in data. If center is None, it is assumed that it is
+            the center of the volume with shape `data.shape[:3]`.
+        roi_radii : int or array-like, optional
+            radii of cuboid ROI in voxels.
+        fa_thr : float
+            FA threshold to compute the WM response function.
+        extract_pam_values : bool, optional
+            Save or not to save pam volumes as single nifti files.
+        sh_order : int, optional
+            Spherical harmonics order used in the CSA fit.
+        odf_to_sh_order : int, optional
+            Spherical harmonics order used for peak_from_model to compress
+            the ODF to spherical harmonics coefficients.
+        parallel : bool, optional
+            Whether to use parallelization in peak-finding during the
+            calibration procedure.
+        num_processes : int, optional
+            If `parallel` is True, the number of subprocesses to use
+            (default multiprocessing.cpu_count()). If < 0 the maximal number
+            of cores minus ``num_processes + 1`` is used (enter -1 to use as
+            many cores as possible). 0 raises an error.
+        gm_response : float, optional
+            Mean diffusivity for GM compartment. If `None`, then grey
+            matter volume fraction is not computed.
+        csf_response : float, optional
+            Mean diffusivity for CSF compartment. If `None`, then CSF
+            volume fraction is not computed.
+        n_iter : int, optional
+            Number of iterations for fODF estimation. Must be a positive int.
+        recon_type : {'smf', 'sos'}, optional
+            MRI reconstruction method: spatial matched filter (SMF) or
+            sum-of-squares (SoS). SMF reconstruction generates Rician noise
+            while SoS reconstruction generates Noncentral Chi noise.
+        n_coils : int, optional
+            Number of coils in MRI scanner -- only relevant in SoS
+            reconstruction. Must be a positive int. Default: 1
+        R : int, optional
+            Acceleration factor of the acquisition. For SIEMENS,
+            R = iPAT factor. For GE, R = ASSET factor. For PHILIPS,
+            R = SENSE factor. Typical values are 1 or 2. Must be a positive
+            integer.
+        voxelwise : bool, optional
+            If true, performs a voxelwise fit. If false, performs a global fit
+            on the entire brain at once. The global fit requires a 4D brain
+            volume in `fit`.
+        use_tv : bool, optional
+            If true, applies total variation regularization. This only takes
+            effect in a global fit (`voxelwise` is set to `False`). TV can only
+            be applied to 4D brain volumes with no singleton dimensions.
+        sphere : Sphere, optional
```

Comment:
not sure what `Sphere`means for the user.  sphere_name so `str` ?

Reply:
You're right, my mistake, I changed it to a `sphere_name`, `str` and defaulted to `repulsion724` in f4e10d3.